.github/workflows/ci.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ 'master' ]
   pull_request:
-    branches: [ 'master' ]
```

Comment:
should we un-delete this?

Reply:
this will allow CI checks to run on all branches (no matter the base branch)